api/src/main/java/io/strimzi/api/kafka/model/kafka/entityoperator/EntityOperatorSpec.java

```diff
@@ -58,6 +59,8 @@ public void setUserOperator(EntityUserOperatorSpec userOperator) {
         this.userOperator = userOperator;
     }
 
+    @Deprecated
+    @DeprecatedProperty(description = "TLS sidecar was deprecated in Strimzi 0.41.0.")
```

Comment:
You should probably use something like this to make it clear it has no meaning now.

```suggestion
    @DeprecatedProperty(description = "TLS sidecar was removed in Strimzi 0.41.0. This property is ignored.")
```